crates/wit-component/tests/interfaces/type-alias.wat

```diff
@@ -35,10 +40,10 @@
           (export (;1;) (interface "foo:foo/foo") (instance (type 1)))
         )
       )
-      (export (;0;) (interface "foo:foo/my-world") (component (type 1)))
+      (export (;1;) (interface "foo:foo/my-world") (type (eq 0)))
```

Comment:
Could this stay as `(component (type ...))` instead of a `(type (eq ..))`? Basically mirror the instance change for interfaces onto worlds too